workspaces/geor/automated_test/sdi/featuretype.xml

```diff
@@ -0,0 +1,54 @@
+<featureType>
+  <id>FeatureTypeInfoImpl-2b665931:15a1d6fbd50:-7ff6</id>
+  <name>sdi</name>
```

Comment:
I would have named it `public_layer` (because later, we'll add `protected_layer`, etc)